src/cluster_linearize.h

```diff
@@ -529,7 +529,19 @@ std::vector<FeeFrac> ChunkLinearization(const DepGraph<SetType>& depgraph, std::
  *            satisfied. Thus, the state being optimal is more a "the eventual output is *known*
  *            to be optimal".
  *
- *            The algorithm terminates whenever an optimal state is reached.
+ * - minimal: We say the state is minimal when it is:
+ *            - acyclic
+ *            - topological, except that inactive dependencies between equal-feerate chunks are
+ *              allowed as long as they do not form a loop.
+ *            - like optimal, no active dependencies whose top feerate is strictly higher than
+ *              the bottom feerate is allowed.
+ *            - no chunk contains a proper non-empty subset which includes all its own in-chunk
```

Comment:
ccd928787cd7f948e4e0d4e8fdf240efbabe14c4

nit: also worth noting in some proper location that minimality also implies connectedness